lib/Lifters/ValueLifter.cpp

```diff
@@ -352,6 +352,13 @@ ValueLifterImpl::Lift(llvm::ArrayRef<uint8_t> data, llvm::Type *type,
       return llvm::ConstantFP::get(type, val.bitsToDouble());
     }
 
+    case llvm::Type::X86_FP80TyID: {
```

Comment:
- [ ] Rebase and get rid of this before merging. This is in a separate PR, I'm just bringing it in for testing purposes.